content/en/security/cloud_security_management/setup/windows.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Setting Up Cloud Security Management On Windows
```

Comment:
```suggestion
title: Setting up Cloud Security Management on Windows
```
I feel like prepositions aren't usually capitalized in docs titles, but couldn't find anything in the style guide about it.